### ISSUE TITLE
Missing Type Definitions

### ISSUE THREAD

**Issue by jdrucey** [headzoo/react-moment#61]
Looks like `subtract` and `add` are missing from the TypeScript definitions file.

So these features cant be used in a TS project without throwing errors.

(Great project BTW, appreciate the work)

**Comment by jdrucey** [comment]
Adding the following to the end of the MomentProps interface solved it for me. i have done it in my own  local override.
```subtract?: {
      years?: number
      quarters?: number
      months?: number
      weeks?: number
      days?: number
      hours?: number
      minutes?: number
      seconds?: number
      milliseconds?: number
    }
    add?: {
      years?: number
      quarters?: number
      months?: number
      weeks?: number
      days?: number
      hours?: number
      minutes?: number
      seconds?: number
      milliseconds?: number
    }
```
Would be neater to create a new addition/subtraction type to avoid the duplication though.
Let me know if you'd like a pull request

**Comment by headzoo** [comment]
Thanks. I made your suggested changes and published v0.7.6.
